src/OrleansProviders/Streams/Common/PooledCache/PooledQueueCache.cs

```diff
@@ -63,15 +65,7 @@ public class PooledQueueCache<TQueueMessage, TCachedMessage>: IPurgeObservable<T
         /// <summary>
         /// Cached message count
         /// </summary>
-        public int ItemCount { get
-            {
-                int count = 0;
-                foreach (var block in this.messageBlocks)
-                {
-                    count += block.ItemCount;
-                }
-                return count;
-            }
+        public int ItemCount { get { return this.itemCount; }
```

Comment:
I will look it up when I got to the office. I guess something is using it since it is public before.